specification/search/resource-manager/Microsoft.Search/stable/2015-08-19/search.json

```diff
@@ -233,7 +233,7 @@
         "x-ms-pageable": {
           "nextLinkName": null
         },
-        "operationId": "QueryKeys_ListBySearchService",
+        "operationId": "QueryKeys_ListBySearchService_Get",
```

Comment:
I'm not sure how AutoRest will handle the underscore before `_Get`. If it doesn't do something sensible, just remove it.